yt_dlp/extractor/adn.py

```diff
@@ -71,10 +71,10 @@ class ADNIE(ADNBaseIE):
         },
         'skip': 'Only available in French and German speaking Europe',
     }, {
-        'url': 'http://animedigitalnetwork.com/video/blue-exorcist-kyoto-saga/7778-episode-1-debut-des-hostilites',
+        'url': 'https://animationdigitalnetwork.com/video/1188-my-deer-friend-nokotan/26192-folge-1',
```

Comment:
The old url was testing the match for `animedigitalnetwork.com` rather than `animationdigitalnetwork.com`. Does the `anime` domain still work?

Reply:
If not, we can remove this test and simplify the `_VALID_URL` pattern

Reply:
The old url was animedigitalnetwork.**fr** but when they switched to a universal .com domain, the animedigitalnetwork.com was not registered. The (already for a few years) deprecated domain still redirects (for now), but is no longer in use.

Reply:
ah gotcha. so we could just do this for now:
```suggestion
        'url': 'https://animedigitalnetwork.fr/video/1188-my-deer-friend-nokotan/26192-folge-1',
```

Reply:
That wouldn't match anymore :face_with_diagonal_mouth: I just saw that animedigitalnetwork.com does match as a valid url, which is what should probably be changed instead

Reply:
oh yep. we can just remove the `only_matching` test then, it's no longer has value

Reply:
:+1: